xpk.py

```diff
@@ -1278,6 +1279,10 @@ def add_env_config(args):
                        'XLA_FLAGS.')
     env['XLA_FLAGS'] = '--xla_dump_to=/tmp/xla_dump/'
 
+  if tensorboard_config:
+    env['UPLOAD_DATA_TO_TENSORBOARD'] = True
+  for key, value in tensorboard_config.items():
+    env[key.upper()] = value
```

Comment:
nit to move this into the if statement (`if tensorboard_config`).

Though I don't think it changes the code path, it groups the related code together.